model/distro/db.go

```diff
@@ -83,6 +83,11 @@ func FindOne(query db.Q) (Distro, error) {
 	return d, db.FindOneQ(Collection, query, &d)
 }
 
+// FindOneId returns one Distro by Id.
+func FindOneId(id string) (Distro, error) {
```

Comment:
Sorry I meant to use FindByID which already exists and correctly returns nil, nil if the item doesn't exist. (Unless returning a pointer causes a lot of issues but then we should make a ticket to get rid of this function bc it doesn't fit convention, and we should call it something like FindOneIdWithoutPointer)